hack/common

```diff
@@ -188,6 +188,34 @@ function push_image() {
     skopeo copy --dest-tls-verify=false docker-daemon:"$1" docker://"$2"
 }
 
+# debug_print [max_count [interval]]
+function debug_print () {
+  local filename=$(basename "$0")
+  local max=${1:-120}
+  local interval=${2:-10}
+  while [ $max -gt 0 ];
+  do
+    clopod=$( oc -n ${NAMESPACE} get pods -o name | egrep "cluster-logging-operator" ) || :
+	clo=$( echo $clopod | awk -F'/' '{print $2}' )
+    if [ -n "$clo" ]; then
+        date >> $ARTIFACT_DIR/$filename.clo.log || :
+        oc -n ${NAMESPACE} logs $clo >> $ARTIFACT_DIR/$filename.clo.log || :
+        echo "-------------------------------------------------" >> $ARTIFACT_DIR/$filename.clo.log || :
+        date >> $ARTIFACT_DIR/$filename.clo.images || :
+        oc -n ${NAMESPACE} exec $clo -- env | egrep _IMAGE >> $ARTIFACT_DIR/$filename.clo.images || :
+        echo "-------------------------------------------------" >> $ARTIFACT_DIR/$filename.clo.images || :
+    fi
```

Comment:
else - something has gone very wrong - maybe do an `oc -n ${NAMESPACE} get pods` or `get deployments`?

Reply:
@jcantrill @JAORMX We can extend this shell function to dump all of the pod logs during the test, rather than figuring out how to do `oc logs`, `oc exec $pod -- logs` etc. in the golang cleanup routine.

Reply:
> @jcantrill @JAORMX We can extend this shell function to dump all of the pod logs during the test, rather than figuring out how to do `oc logs`, `oc exec $pod -- logs` etc. in the golang cleanup routine.

Let me open another pr for this matter since this debug print is not currently used in this pr.  I temporarily enabled it for the debugging legacy e2e in the CI tests.

Reply:
As in this conversation, improving the debug print will be done in the separate pr.